components/scripts/lib/build-scan-online.sh

```diff
@@ -101,5 +101,5 @@ fetch_build_scan_data() {
     args+=( "${run_num},${build_scan_urls[run_num]}" )
   done
 
-  invoke_java "${FETCH_BUILD_SCAN_DATA_JAR}" "${args[@]}"
+  JAVA_HOME="${CLIENT_JAVA_HOME:-$JAVA_HOME}" invoke_java "${FETCH_BUILD_SCAN_DATA_JAR}" "${args[@]}"
```

Comment:
This line is a small bit of idiomatic bash to say:

Set JAVA_HOME equal to the value of CLIENT_JAVA_HOME.  If CLIENT_JAVA_HOME is not set or has no value, then set JAVA_HOME equal to the value of the JAVA_HOME value.

It basically enables the fallback to JAVA_HOME if CLIENT_JAVA_HOME is not set.